app/Config/core.php

```diff
@@ -254,20 +254,20 @@
  *
  * To use, prefix your JavaScript link URLs with '/cjs/' instead of '/js/' or use JsHelper::link().
  */
-	//Configure::write('Asset.filter.js', 'custom_javascript_output_filter.php');
+    //Configure::write('Asset.filter.js', 'custom_javascript_output_filter.php');
 
 /**
  * The class name and database used in CakePHP's
  * access control lists.
  */
-	Configure::write('Acl.classname', 'DbAcl');
-	Configure::write('Acl.database', 'default');
+    Configure::write('Acl.classname', 'DbAcl');
+    Configure::write('Acl.database', 'default');
 
 /**
  * Uncomment this line and correct your server timezone to fix
  * any date & time related errors.
  */
-	//date_default_timezone_set('UTC');
+    //date_default_timezone_set('UTC');
```

Comment:
__[staging][code_sniffer]__
* No space found before comment text; expected "// date_default_timezone_set('UTC');" but found "//date_default_timezone_set('UTC');"
* Inline comments must start with a capital letter
* Inline comments must end in full-stops, exclamation marks, or question marks
* There must be no blank line following an inline comment